templates/layout-slider.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Netzstrategen\Gallerya;
 
-$show_navigation = count($images) > 5;
+$show_navigation = count($images) > $min_image_to_show_nav;
```

Comment:
I would expect the navigation to be shown by a minimum of my filtered value. But indeed it's minimum the value +1. So we should change this condition to `>=`.